### DISCUSSION TITLE
react-text-mask, controller not update

### DISCUSSION THREAD

**Discussion by SergeyWebDeveloper** [react-hook-form/react-hook-form#3479] [Q&A]
Hello. Please tell me, when the value comes from the API, the value in the controller does not change according to the mask, only when you start to enter with your hands, then everything is fine.
Example - https://codesandbox.io/s/react-hook-form-controller-template-forked-y33i6?file=/src/index.js

**Comment by bluebill1049** [comment]
I don't think this component works with props as value, you may want to try a different package?

**Reply by SergeyWebDeveloper** [reply]
Unfortunately no
